public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -100,208 +97,213 @@ code-example(language="sh" class="code-shell").
+    Then add the `ngIf` built-in directive and set it to the `selectedHero` property of the component.
 
-    We wrap the HTML hero detail content of our template with a `<div>`.
-    Then we add the `ngIf` built-in directive and set it to the `selectedHero` property of our component.
+  +makeExample('toh-2/ts/app/app.component.1.html', 'ng-if', 'app.component.ts (ngIf)')(format='.')
 
-  +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'ng-if', 'app.component.ts (ngIf)')
+  :marked
+    The list of names displays again in the browser.
 
   .alert.is-critical
     :marked
       Remember that the leading asterisk (`*`) in front of `ngIf` is
       a critical part of this syntax.
+  //  CF: is a "critical" type note appropriate here? Could I merge this sentence with
+    the next section?
   :marked
     When there is no `selectedHero`, the `ngIf` directive removes the hero detail HTML from the DOM.
-    There will be no hero detail elements and no bindings to worry about.
+    There are no hero detail elements or bindings to worry about.
 
     When the user picks a hero, `selectedHero` becomes "truthy" and
     `ngIf` puts the hero detail content into the DOM and evaluates the nested bindings.
+  //  CF: Is "truthy" jargon? How about: "`selectedHero` is set to true ..."
+
   .l-sub-section
     :marked
-      `ngIf` and `ngFor` are called “structural directives” because they can change the
+      `ngIf` and `ngFor` are called "structural directives" because they can change the
       structure of portions of the DOM.
       In other words, they give structure to the way Angular displays content in the DOM.
 
-      Learn more about `ngIf`, `ngFor` and other structural directives in the
-      [Structural Directives](../guide/structural-directives.html) and
-      [Template Syntax](../guide/template-syntax.html#directives) chapters.
+      Read more about `ngIf`, `ngFor`, and other structural directives in the
+      [Structural Directives](../guide/structural-directives.html) page and the
+      [Built-in directives](../guide/template-syntax.html#directives) section of the
+      [Template Syntax](../guide/template-syntax.html) page.
+
 
   :marked
-    The browser refreshes and we see the list of heroes but not the selected hero detail.
-    The `ngIf` keeps it out of the DOM as long as the `selectedHero` is undefined.
-    When we click on a hero in the list, the selected hero displays in the hero details.
-    Everything is working as we expect.
+    When the browser refreshes, the list of heroes displays, but the selected hero detail does not.
+    The `ngIf` keeps the selected hero detail out of the DOM as long as the `selectedHero` is undefined.
+    When you click a hero in the list, the selected hero displays in the hero details.
+
+    ### Style the selection
 
-    ### Styling the selection
+    While the selected hero displays in the details area, it's difficult to quickly locate that hero in the list above.
```

Comment:
I think the following is a bit more clear
>  While the selected hero's details are displayed _below the list_, it's difficult to identify the selected hero within the list itself.

I'd like that to be a stand-alone paragraph before beginning 
>To fix that